website/docs/guides/1.0.0-upgrade-guide.html.markdown

```diff
@@ -230,28 +230,120 @@ google_cloud_kms_config {
+
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.0.id
+      frequency_interval = 1
+      frequency_type     = "hourly"
+      retention_unit     = "days"
+      retention_value    = 1
+    }
+
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.1.id
+      frequency_interval = 1
+      frequency_type     = "daily"
+      retention_unit     = "days"
+      retention_value    = 2
+    }
+
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.2.id
+      frequency_interval = 4
+      frequency_type     = "weekly"
+      retention_unit     = "weeks"
+      retention_value    = 3
+    }
+
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.3.id
+      frequency_interval = 5
+      frequency_type     = "monthly"
+      retention_unit     = "months"
+      retention_value    = 4
+    }
+  }
+}
+```
+
+New configuration
+```hcl
+resource "mongodbatlas_cloud_backup_schedule" "test" {
+  project_id   = mongodbatlas_cluster.my_cluster.project_id
+  cluster_name = mongodbatlas_cluster.my_cluster.name
+
+  reference_hour_of_day    = 3
+  reference_minute_of_hour = 45
+  restore_window_days      = 4
+  
+  // This will now add the desired policy items to the existing mongodbatlas_cloud_backup_schedule resource
+  policy_item_hourly {
+    frequency_interval = 1
+    retention_unit     = "days"
+    retention_value    = 1
+  }
+  policy_item_daily {
+    frequency_interval = 1
+    retention_unit     = "days"
+    retention_value    = 2
+  }
+  policy_item_weekly {
+    frequency_interval = 4
+    retention_unit     = "weeks"
+    retention_value    = 3
+  }
+  policy_item_monthly {
+    frequency_interval = 5
+    retention_unit     = "months"
+    retention_value    = 4
+  }
+
+}
+```
+
+The following changes are:
+    1. The IDs for policy and policies items are not necessary to set anymore.
+    2. We separated the policies items into a single object for policy item `hourly`, `daily`, `weekly` and `monthly`
+    3. The `frequency_type` for a policy item is not necessary to be set.
+
+2. Remove your existing `mongodbatlas_cloud_provider_snapshot_backup_policy` resources from terraform state
    **NOTE** before doing any terraform state change please do a backup
 
 ```bash
 $ terraform state rm mongodbatlas_cloud_provider_snapshot_backup_policy.test
 ```
 
-2. Change the version of provider to the v1.0.0
+3. Change the version of provider to the v1.0.0
+
+4. In order to import the new resource, it needs to change the configuration file from the `mongodbatlas_cloud_provider_snapshot_backup_policy` config to `mongodbatlas_cloud_backup_schedule` with the example shared in the first step.
```

Comment:
Slight edit to help readability and please add a link to the new doc page where I call out (LINK)(/LINK) below:
 4. In order to import the new resource, you must first change your Terraform configuration file, replacing`mongodbatlas_cloud_provider_snapshot_backup_policy` with `mongodbatlas_cloud_backup_schedule`.   See the example shared above or find more information in the (LINK)`mongodbatlas_cloud_backup_schedule` resource documentation (/LINK)